### PR TITLE
fixed nonce issue and basestring variable

### PR DESCRIPTION
**Pull request by punjabisingh**
# Description

Fixed the bug on nonce variable to return base64string from long and fixed the issue for basestring to have null variable. 

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Run Unit Test Cases.

### CODE REVIEW

File: ApiUtilLib/ApiAuthorization.cs [hunk 1]
```diff
@@ -220,23 +220,26 @@ public static long NewTimestamp()
...
                 // Buffer storage.
-                byte[] data = new byte[8];
+                data = new byte[8];
 
                 // Fill buffer.
                 rng.GetBytes(data);
 
                 // Convert to int 64.
-                nonce = BitConverter.ToInt64(data, 0);
+                //nonce = BitConverter.ToInt64(data, 0);
```
Comment (robincher) [inline ApiUtilLib/ApiAuthorization.cs]: Can removed unused code

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,3 +3,5 @@
 ## Added 
 ### V1.0-SNAPSHOT
 + 2017-10-29 - Initial release.
+### V1.1-SNAPSHOT
++ 2018-09-10 - Bug fixes.
```
Comment (robincher) [inline CHANGELOG.md]: Can be more precise on the changes, like what is done to enhance the library to handle query params.

**Review by robincher — CHANGES_REQUESTED** [review]
Require some minor updates and it will be good to go

**Comment by punjabisingh** [comment]
updated bug fixes

**Pull request closed without merging**